dts/bindings/interrupt-controller/nxp,nbu.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP NBU interruption information
+
+include: base.yaml
+
+compatible: "nxp,nbu"
+
+properties:
+  interrupts:
+    required: true
+  nbu-name:
+    default: "NBU NXP"
+    type: string
+    description: Name of the Narrow Band Unit interruption initialization
```

Comment:
maybe I am missing something, but I'm not seeing what this is used for, should we be using the label property as per DT specification? it depends on what this string is used for